spotlight-client/src/DataStore/UserStore.ts

```diff
@@ -121,6 +130,10 @@ export default class UserStore {
           this.isAuthorized = false;
           this.awaitingVerification = true;
         }
+        if (stateCode && this.rootStore) {
```

Comment:
I'm setting the tenantStore's `locked` and `currentTenantId` here, does that sound alright? I see we also set the `currentTenantId` in `withRouteSync`

Reply:
Revised this and now we're just setting `currentTenantId` here. I see that `withRouteSync` and the tenantStore constructor all could change the `currentTenantId`